docker-images/development/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM strimzi/base:latest
```

Comment:
Does this file belong here?

Reply:
Sorry, leftover from my work with strimzi-cli image. Going to remove it.